### ISSUE TITLE
[v2] Unable to download files over ~1 Mb

### ISSUE THREAD

**Issue by wasd171** [telepresenceio/telepresence#1773]
Hi, I seem to have an issue with telepresence v2.3.0 on macOS 11.3. This issue was not present with telepresence v2.2.2

My service is exposed via ingress-nginx on domain konstantin.portal.careerlunch.com
I am using telepresence to forward traffic to my local Docker container
```
telepresence intercept deploymentName --port randomPort:80 --docker-run imageName
```

When accessing this server through domain, some requests are not being finished, they only terminate after 1 minute timeout. It seems to happen for all larger files (over ~1 Mb). This can be seen on a first screenshot
<img width="1304" alt="tele2-domain" src="https://user-images.githubusercontent.com/8266564/120784488-ce987000-c534-11eb-864d-12652447054e.png">

However, accessing this server through localhost works fine (second screenshot)
<img width="1304" alt="tele2-localhost" src="https://user-images.githubusercontent.com/8266564/120786608-17e9bf00-c537-11eb-9f59-b151be0c6f32.png">

Everything works as expected when this container is deployed via helm, so it's probably not an ingress-nginx issue, seems like tele2 is unable to return the whole file (nginx logs):
```
2021/06/04 09:31:12 [error] 10036#10036: *24811860 upstream timed out (110: Operation timed out) while reading upstream
```

**Comment by josecv** [comment]
@wasd171 could I ask a favour? Please try [setting the `rootDaemon` log level to debug](https://www.getambassador.io/docs/telepresence/latest/reference/config/#laptop-side-configuration), reproducing the issue, and attaching the `daemon.log` file to this ticket. You can find the log file under `~/Library/Logs/telepresence` on MacOS or `~/.cache/telepresence/logs` on Linux

**Comment by efunk** [comment]
Delivered in the Telepresence 2.3.1 GA release.

**Comment by efunk** [comment]
Resolved in Telepresence 2.3.2 GA
